lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -395,7 +383,7 @@ export class UserAgentApplication {
 
     // If already in progress, do not proceed
     if (this.acquireTokenInProgress) {
-      this.errorReceivedCallback(ClientAuthError.createAcquireTokenInProgressError(), this.getAccountState(this.silentAuthenticationState));
+      this.authResponseCallback(ClientAuthError.createAcquireTokenInProgressError(), buildResponseStateOnly(this.getAccountState(this.silentAuthenticationState)));
```

Comment:
so the return is (err, resp) ?

Reply:
have we updated docs with this usage

Reply:
No not yet, presenting the PR first, we will update docs before i merge this.

Reply:
Yes that is correct for the function signature

Reply:
What is the impact on graph and railyard?

Reply:
and the pax team?

Reply:
It will be a breaking change, we discussed with state as well. We can take this discussion offline and merge this PR later if needed. Resolving for now.